Makefile

```diff
@@ -135,6 +135,7 @@ $(GEN_TIMESTAMP): $(shell find apis -name '*.go')  $(OPERATOR_SDK) $(CONTROLLER_
 	@$(CONTROLLER_GEN) object paths="./apis/..."
 	@$(CONTROLLER_GEN) crd:crdVersions=v1 rbac:roleName=clusterlogging-operator paths="./..." output:crd:artifacts:config=config/crd/bases
 	@bash ./hack/generate-crd.sh
+	echo 'package version; var Version = "$(or $(CI_CONTAINER_VERSION),$(LOGGING_VERSION).0)"' > version/version.go
```

Comment:
what happens when these variables are not available?

Reply:
@alanconway could you please guide here. Jeff is asking about situations when CI_CONTAINER_VERSION and LOGGING_VERSION is not made available. Is that could be a situation in general in CI pipeline ? We also have env variable through which we are setting up version information here. I have made both provision in this PR.

Reply:
@pmoogi-redhat This target AFAIK does not get run during CI. Regardless it is always good practice to provide a default otherwise this step out right fails when doing local development